src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -556,6 +571,31 @@ export function getEventLineChartVisualDef(visualId: string, viewName: string, t
   return visualDef;
 }
 
+export function getPathAnalysisChartVisualDef(visualId: string, viewName: string) : Visual {
+
+  const visualDef = JSON.parse(readFileSync(join(__dirname, './templates/path-analysis-chart.json')).toString()) as Visual;
```

Comment:
How about replacing the placeholders in template json by mustache instead of below code snippet?

Reply:
fixed in #178

Reply:
fixed